src/com/android/volley/toolbox/ImageLoader.java

```diff
@@ -180,7 +180,27 @@ public void onResponse(ImageContainer response, boolean isImmediate) {
             public void onErrorResponse(VolleyError error) {
 
             }
-        }, 0, 0, false, true);
+        }, 0, 0, false, true, 0);
+    }
+
+    /**
+     * get Image for background caching
+     * @param requestUrl
+     * @param ttl allow a custom ttl for caching
+     * @return
+     */
+    public ImageContainer getImageBackground(String requestUrl, long ttl) {
```

Comment:
Can we chain this overloaded method with the existing `getImageBackground` instead of having separate impl?